src/core/views/widget.js

```diff
@@ -191,6 +194,12 @@ class Widget extends Component {
     })
   }
 
+  handleFormScreenChanged (event) {
```

Comment:
What about popup embed type? I think it would be beneficial to have this event for both widget and embed.

Reply:
Hi @mathio, thanks for the feedback. I just added the same functionality to the popup as well.